src/_content/posts/announcing-lineup.md

```diff
@@ -0,0 +1,62 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2025-07-14 00:00:00
```

Comment:
```suggestion
published_datetime: 2025-07-17 18:00:00
```

Getting this up to date so we can merge this today.

@PeterGrand want to give this one final look before merging?